### PR TITLE
LRAC-7405  Allow send custom events in JS client

### PR DESCRIPTION
**Pull request by fcodias**
https://issues.liferay.com/browse/LRAC-7405

### CODE REVIEW

**Comment by fcodias** [comment]
ci:test:sf

File: modules/apps/analytics/analytics-client-js/src/analytics.js [hunk 1]
```diff
@@ -241,30 +242,47 @@ class Analytics {
...
+	/**
+	 * Registers an event that is to be sent to Analytics Cloud
+	 * @param {string} eventId Id of the event
+	 * @param {string} applicationId ID of the application that triggered the event
+	 * @param {Object} eventProps Complementary information about the event
+	 */
+	send(eventId, applicationId, eventProps) {
+		if (
+			this._isTrackingDisabled() ||
+			!applicationId ||
```
Comment (andreldm) [inline modules/apps/analytics/analytics-client-js/src/analytics.js:277]: Only checking for `!applicationId` isn't enough?

Comment (andreldm) [inline modules/apps/analytics/analytics-client-js/src/analytics.js]: Only checking for `!applicationId` isn't enough?

File: modules/apps/analytics/analytics-client-js/test/manual/blogs.html [hunk 2]
```diff
@@ -80,7 +80,8 @@ <h3>1914 translation by H. Rackham</h3>
 				Analytics.create({
 					channelId: '4321',
 					dataSourceId: '1234',
-					endpointUrl: 'https://osbasahcerebropublisher-asahlfr.lfr.io'
+					endpointUrl: 'http://127.0.0.1:8084',
+					projectId: 'osb-asah'
```
Comment (andreldm) [inline modules/apps/analytics/analytics-client-js/test/manual/blogs.html]: The default project id is `osbasah`, no dash.

**Review by andreldm — APPROVED** [review]
Besides these two remarks, looks good to me :smiley:

**Comment by fcodias** [comment]
ci:test:sf

**Review by Masterfoni — APPROVED** [review]
LGTM

**Comment by fcodias** [comment]
ci:forward

**Pull request closed without merging**